sycl/include/CL/sycl/INTEL/esimd/esimd_enum.hpp

```diff
@@ -104,6 +104,14 @@ enum class CacheHint : uint8_t {
   ReadInvalidate = 5
 };
 
+enum class EsimdSbarrierType : uint8_t {
+  ESIMD_SBARRIER_WAIT = 0,  // split barrier wait
```

Comment:
I don't think ESIMD_SBARRIER prefix is needed in enum values, as it is embedded into the class name.

Also, I would suggest to use `EsimdSbarrier` instead of `EsimdSbarrierType`: `EsimdSbarrier::WAIT` bears all the relevant semantic info. In `EsimdSbarrierType::ESIMD_SBARRIER_WAIT` 'Esimd' and 'barrier' are repeated twice.

Reply:
Updated.